Author: aurel32
Status: in BTS
Note: needs and update of config.{guess,sub}.

--- agsync-0.2-pre.orig/mal-files/AGNet.h
+++ agsync-0.2-pre/mal-files/AGNet.h
@@ -61,7 +61,7 @@
 #               include <sys/filio.h>
 #               include <arpa/inet.h>
 #           else
-#               if defined(__FreeBSD__) || defined(_HPUX_SOURCE)
+#               if defined(__FreeBSD__) || defined(__FreeBSD_kernel__) || defined(_HPUX_SOURCE)
 #                   include <sys/ioctl.h>
 #                   include <arpa/inet.h>
 #               else
